### PR TITLE
faq page added, little change required

### PR DESCRIPTION
**Pull request by 97yogitha**
@rohitvarkey  faq page is created. I need to attach 
[Application_for_Transcripts.pdf](https://github.com/IE-NITK/NITK-Student-Council-Website/files/139143/Application_for_Transcripts.pdf) file in "how to get address proof " section. How should it be done?

### CODE REVIEW

**Comment by rohitvarkey** [comment]
@97yogitha Maybe something like [this](https://docs.djangoproject.com/en/dev/ref/request-response/#telling-the-browser-to-treat-the-response-as-a-file-attachment)?

@abhijithanilkumar Can you help out with this?

**Comment by jaidevanoop** [comment]
@97yogitha By default, keep the FAQs closed. @rohitvarkey wanted it numbers as well

**Comment by rohitvarkey** [comment]
@97yogitha Is this ready to be merged? Also what fonts have you used in this?

File: src/static/site/css/faq.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.collapsible-header {
+	height:40px;
+	font-size:2em;
+
+}
+div {
+	font-family: GillSans;
```
Comment (rohitvarkey) [inline src/static/site/css/faq.css]: Do we need this? Shouldn't we keep the font's same throughout?

Comment (97yogitha) [inline src/static/site/css/faq.css]: okay. I'll change it. I was just trying out  fonts.

Comment (97yogitha) [inline src/static/site/css/faq.css]: @rohitvarkey  do you want me to add anything else??

Comment (rohitvarkey) [inline src/static/site/css/faq.css]: Nope. just remove the font css and it should be fine. Also close the questions by default!

File: src/templates/faq.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+<ul class="collapsible" data-collapsible="expandible">
+    <li>
+      <div class="collapsible-header active">How to attain address proof of NITK?</div>
+      <div class="collapsible-body">
+      	
+      		<p>Go to academic section, apply for bonafide.
+      		<br>
+      		Fill out the <a href="{% static 'site/Application_for_Transcripts.pdf' %}">this </a>form.
+      		<br>
+      		Pay Rs200 per transcript at the cash section
```
Comment (rohitvarkey) [inline src/templates/faq.html]: Rs. 200

File: src/templates/faq.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+      		Collect your transcripts in 10-15 days</p>
+      </div>
+    </li>
+
+ </ul>
+ <ul class="collapsible" data-collapsible="expandible">
+    <li>
+      <div class="collapsible-header active">How to book the guest house?</div>
+      <div class="collapsible-body">
+		<p>DFW office, AC corridor, get gorm, fill and get sign from HoD/Fac AD, give it back to DFW office.
```
Comment (rohitvarkey) [inline src/templates/faq.html]: *form. And expand on DFW the first time you use it.

**Comment by rohitvarkey** [comment]
@97yogitha Please finish this off! It needs a rebase too.

**Pull request merged**